README.md

```diff
@@ -246,9 +244,9 @@ python examples/run_smarts.py --algo SAC --scenario ./scenarios/loop --n_agents
 If you're comfortable using docker or are on a platform without suitable support to easily run SMARTS (e.g. an older version of Ubuntu) you can run the following,
 
 ```bash
-$ cd /path/to/SMARTS
+$ cd </path/to/SMARTS>
 $ docker run --rm -it -v $PWD:/src -p 8081:8081 huaweinoah/smarts:<version>
```

Comment:
I found talking with the waymo integration team that there is a requirement that the repository be set up as editable previously or afterwards. This may need a `pip install -e .` from inside or outside the container to make sure the repository is editable and at the right version. 

Using `pip install -e .` though does not work without internet access though which means extra work to get pip pointed to the right location or proxy setup requirements.